app/scripts/contentscript.js

```diff
@@ -81,6 +82,83 @@ function createCommitChart(){
     myChart.setOption(option)
 }
 
+/**
+ * Create latel to pull request interaction graph
+ */
+const createLabel = (score) => {
+    return {
+        normal: {
+        formatter: '{a|{b}}{abg|}\n{hr|}\n{b|Score:  ' + score + '}  {per|{d}%}  ',
+        backgroundColor: '#eee',
+        borderColor: '#aaa',
+        borderWidth: 1,
+        borderRadius: 4,
+        rich: {
+            a: {
+                color: '#586069',
+                lineHeight: 22,
+                align: 'center',
+                fontSize: 14
+            },
+            hr: {
+                borderColor: '#aaa',
+                width: '100%',
+                borderWidth: 0.5,
+                height: 0
+            },
+            b: {
+                color: '#586069',
+                fontSize: 14,
+                lineHeight: 22
+            },
+            per: {
+                color: '#eee',
+                backgroundColor: '#334455',
+                padding: [2, 4],
+                borderRadius: 2,
+                lineHeight: 22
+            }
+        }
+    }
+}
+}
+
+/**
+ * Create pull request graph
+ */
+const createPullRequestChart = () => {
+    var content = document.getElementsByClassName("new-discussion-timeline experiment-repo-nav")
+    var repoContent = document.getElementsByClassName("repository-content")
+    var node = document.createElement('div')
+    node.innerHTML = graph()
+    content[0].insertBefore(node, repoContent[0])
+    var myChart = echarts.init(document.getElementById('my-graph'))
+    let option = {
+        tooltip: {
+            trigger: 'item',
+            formatter: "{a} <br/>{b}: {c} ({d}%)"
+        },
+        series: [
+            {
+                name:'Pull Request Interaction',
+                type:'pie',
+                radius: ['0%', '55%'],
+                color: ['#e9def5', '#e9d8ff', '#d2beeb', '#bb9ee1', '#a37fd7', '#8a61cc', '#6f42c1'],
+                data:[
+                    {value:335, name:'Old Open without comment', label:createLabel('0')},
+                    {value:310, name:'Refused without comment', label:createLabel('0.1')},
+                    {value:234, name:'Open with old comment', label:createLabel('0.3')},
+                    {value:135, name:'Refused with comment', label:createLabel('0.7')},
+                    {value:1048, name:'Open with recent comment', label:createLabel('0.9')},
+                    {value:251, name:'Merjed without comment', label:createLabel('0.9')},
+                    {value:147, name:'Merjed with comment', label:createLabel('1')}
```

Comment:
The values are hardcoded. The ideal solution is to receive those data as parameters. The names and labels may be hardcoded since the data is going to be received from API in order.

Reply:
Since the data is not available yet I did it this way but I will pass this data by parameters in the method

Reply:
1 - Insert a pading is quite simple, I will do this;
2 - This I had done, but since it was not in the prototype I removed. I'll reinsert

Reply:
fixed